src/progress-indicator/progress-indicator.tsx

```diff
@@ -140,6 +140,7 @@ export const ProgressIndicator = <T,>({
                 <Indicator
                     key={stepIndex}
                     aria-label={getAriaLabel(stepIndex, currentIndex)}
+                    id={getAriaLabel(stepIndex, currentIndex)}
```

Comment:
should use data-testid or className (sorry, not sure what the convention is), to avoid multiple indicators sharing the same ids

Reply:
I also not sure about what the convention lol. But QA need a unique Id, so I add a index